modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -149,31 +169,117 @@ function inope_policy_review_form_submit($form, &$form_state) {
   // Load objects from which their properties will be displayed.
   $account = user_load($form_state['values']['inope_policy_review_hidden_uid']);
   $node = node_load($form_state['values']['inope_policy_review_hidden_nid']);
+  $text = check_plain($form_state['values']['inope_policy_review_changes_requested_textarea']);
 
   // If 'I have reviewed this policy and approved it for use' is selected.
   if ($form_state['values']['inope_policy_review_status_radio'] == 0) {
-    drupal_set_message(t('Policy approved for use'));
-    // TODO: put this into the database.
+
+    $policy_review_id = inope_policy_review_insert_query($node, $account, '', 1);
+
+    if ($policy_review_id != 0) {
+      // Thank the user for their approval.
+      drupal_set_message(
+        t('Thank you @username for approving the policy', array(
+          '@username' => $account->name,
+        )));
+    }
   }
   // If 'I have reviewed this policy and would like to see changes' is selected.
   elseif ($form_state['values']['inope_policy_review_status_radio'] == 1) {
-    drupal_set_message(
-      t('User : @userid : @username submitted the following change request:', array(
-        '@userid' => $account->uid,
-        '@username' => $account->name,
-      )));
 
-    drupal_set_message($form_state['values']['inope_policy_review_changes_requested_textarea']);
+    $policy_review_id = inope_policy_review_insert_query($node, $account, $text, 0);
 
-    drupal_set_message(t('Node ID : @nodeid', array('@nodeid' => $node->nid)));
-    drupal_set_message(t('Node VID : @nodevid', array('@nodevid' => $node->vid)));
-    // TODO: put this into the database.
+    if ($policy_review_id != 0) {
+      // Thank the user for their request.
+      drupal_set_message(
+        t('Thank you @username for submitting the following change request:', array(
+          '@username' => $account->name,
+        )));
+      // Check_plain happened when this variable was declared.
+      drupal_set_message($text);
+    }
   }
   // If a bug is encountered.
   else {
     drupal_set_message(t('Form state not recognized'));
   }
+}
+
+/**
+ * Select query for the current node policy review notes.
+ *
+ * Query the policy review table for any records related to the current
+ * node, note status, and are from the current user.
+ */
+function inope_policy_review_select_query($node, $account) {
+
+  // Select query to get the latest node revision timestamp.
+  $node_revision = db_query("
+  SELECT vid, timestamp FROM {node_revision} WHERE vid = :vid", array(
```

Comment:
I think we forgot to turn on revisions for the policy node -- so we should probably do that via features just so this will work properly.